Bottom: 5beebb3c4794e265e999d03746b9a719a4b5348b
Top:    16645cb11908906950dc0cf78a7ac5283c4a36dc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 19:14:53 +0000

Refresh of patch

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 06a4d7e..70cc51b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -742,6 +742,9 @@ object3d_from_board_outline (void)
 {
   POLYAREA *board_outline = board_outline_poly (true);
   POLYAREA *pa;
+  appearance *board_appearance;
+  appearance *top_bot_appearance;
+  GList *objects;
 
 #if 0
   return object3d_from_soldermask_within_area (board_outline, TOP_SIDE);
@@ -760,10 +763,6 @@ object3d_from_board_outline (void)
     }
   while ((pa = pa->f) != board_outline);
 
-  appearance *board_appearance;
-  appearance *top_bot_appearance;
-  GList *objects;
-
   board_appearance = make_appearance ();
   top_bot_appearance = NULL;
   appearance_set_color (board_appearance,   1.0, 1.0, 0.6);
